### PR TITLE
PGD re-edit group 5

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/pgd/5/durability/limitations.mdx [hunk 1]
```diff
@@ -43,10 +43,10 @@ nodes in a group. If you use this feature, take the following limitations into a
 
 ## Eager
 
-[Eager](../consistency/eager) is available through Group Commit. It avoids conflicts by eagerly aborting transactions that might clash. It's subject to the same limitations as Group Commit.
+[Eager](../consistency/eager) is available through Group Commit. It avoids conflicts by eagerly aborting transactions that might clash. It's subject to the same limitations as Group Commit. In addition:
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/durability/limitations.mdx:46]: There was nothing introducing this single bullet. Is this what was meant? Or can we just take out the bullet  (making it a regular paragraph) and the words "in addition" that I added?

Comment (gvasquezvargas) [reply product_docs/docs/pgd/5/durability/limitations.mdx:46]: @djw-m I think Betsy's edit here makes sense. 
Alternatively, we could make a Note or Important note taking part of the text and the bullet:
```
Important 
Eager is subject to the same limitations as Group Commit. Additionally, it doesn't allow the `NOTIFY` SQL command or the `pg_notify()` function. It also doesn't allow `LISTEN` or `UNLISTEN`.
```

File: product_docs/docs/pgd/5/durability/overview.mdx [hunk 2]
```diff
@@ -41,21 +41,19 @@ Commit scopes allow four kinds of controlling durability of the transaction:
...
--   [PGD Synchronous Commit](synchronous_commit): This kind of commit scope allows for a behaviour where the origin node awaits a majority of nodes to confirm and behaves more like a native Postgres synchronous commit.
+-   [PGD Synchronous Commit](synchronous_commit): This kind of commit scope allows for a behavior where the origin node awaits a majority of nodes to confirm and behaves more like a native Postgres synchronous commit.
 
 
 !!! Note Legacy synchronization availability 
 For backward compatibility, PGD still supports configuring synchronous
 replication with `synchronous_commit` and `synchronous_standby_names`. See
 [Legacy synchronous replication](legacy-sync) for more on this option. We
-recommend users instead use [PGD Synchronous_Commit](synchronous_commit). 
+recommend you instead use [PGD Synchronous Commit](synchronous_commit).
```
Comment (gvasquezvargas) [inline product_docs/docs/pgd/5/durability/overview.mdx:58]: This sounds a bit clunky for me as a non-native, maybe: 
```suggestion
recommend that you use [PGD Synchronous Commit](synchronous_commit) instead. 
```
WDYT?

**Review by gvasquezvargas — COMMENTED** [review]
Left two minor comments.

**Comment by ebgitelman** [comment]
@djw-m @gvasquezvargas @jpe442 Can someone approve this and/or let me know if you prefer Gaby's revision in the comments above?

File: product_docs/docs/pgd/5/durability/limitations.mdx (hunk 1 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/durability/limitations.mdx:46]: Take out the bullet. I think that was a side effect of these being extracted from elsewhere. (And lose the In Addition)

**Comment by ebgitelman** [comment]
@gvasquezvargas I think this is ready to approve.

**Review by gvasquezvargas — APPROVED** [review]
LGTM!

**Pull request merged**